app/assets/javascripts/discourse/app/components/scrolling-post-stream.js

```diff
@@ -103,6 +85,7 @@ export default MountWidget.extend({
     const slack = Math.round(windowHeight * 5);
     const onscreen = [];
     const nearby = [];
+    // body.scrollTop fallback here is for iOS 12 support
     const windowTop =
       document.documentElement.scrollTop || document.body.scrollTop;
```

Comment:
How about using the standard API, so we don't have to handle iOS 12 separately?
```suggestion
    const windowTop = document.scrollingElement.scrollTop;
```

I confirmed it returns `<body>` on iOS12, and `<html>` on modern browsers

Reply:
That's a nice find, I didn't know about `document.scrollingElement`. Let's try it!